Yontech.Fat/WebControls/ITextControl.cs

```diff
@@ -1,18 +1,19 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace Yontech.Fat.WebControls
 {
-    public interface ITextControl : IWebControl
-    {
-        void Click();
-        void Click(int relativeX, int relativeY);
-        string Text { get; }
-        void InnerTextShouldBe(string text);
-        void ShouldContainText(string text);
-        void ShouldNotContainText(string text);
-        void SendKeys(string keys);
-    }
+  public interface ITextControl : IWebControl
+  {
+    void Click();
+    void Click(int relativeX, int relativeY);
+    string Text { get; }
+    void InnerTextShouldBe(string text);
+    void ShouldContainText(string text);
+    void ShouldNotContainText(string text);
+    void SendKeys(string keys);
+    void ClearText();
```

Comment:
Clear text from a label? If you are looking for a textbox than there is a textbox control